modules/node/src/conditionalCommitment/conditionalCommitment.repository.ts

```diff
@@ -45,14 +46,10 @@ export class ConditionalTransactionCommitmentRepository extends Repository<
   ): Promise<ConditionalTransactionCommitment[]> {
     return this.createQueryBuilder("conditional")
       .leftJoinAndSelect("conditional.app", "app")
-      .where(
-        "app.type <> :rejected", { rejected: AppType.REJECTED },
-      )
+      .where("app.type <> :rejected", { rejected: AppType.REJECTED })
```

Comment:
Has this been consolidated with Layne's change to remove the rejected app type? I thought rejected apps were deleted & therefore aren't around to have their AppType set to rejected.

Reply:
That change isn't in staging yet -- lets merge this first since there are a few extra considerations for fixing statuses